dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/main/java/org/apache/dolphinscheduler/plugin/task/mlflow/MlflowConstants.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.mlflow;
+
+public class MlflowConstants {
+    private MlflowConstants() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static final String JOB_TYPE_AUTOML = "AutoML";
+
+    public static final String JOB_TYPE_BASIC_ALGORITHM = "BasicAlgorithm";
+
+    public static final String PRESET_AUTOML_PROJECT = "https://github.com/jieguangzhou/MLflow-AutoML";
+
+    public static final String PRESET_BASIC_ALGORITHM_PROJECT = "https://github.com/jieguangzhou/mlflow_sklearn_gallery";
```

Comment:
I find you use your repo in our codebase, why?

Reply:
The MLflow can run the GitHub URL as an MLflow project. In this implementation, if we wanted to provide the preset algorithm capability for the MLFlow component for my project, we needed a repository to hold the code.

I don't know how these repos should be created, do you have any suggestions?

Reply:
> The MLflow can run the GitHub URL as an MLflow project. In this implementation, if we wanted to provide the preset algorithm capability for the MLFlow component for my project, we needed a repository to hold the code.
> 
> I don't know how these repos should be created, do you have any suggestions?

We'd better not to use personal repository here, is `PRESET_BASIC_ALGORITHM_PROJECT` is required in this task? If not we can just remove it, if it's required, we'd need to create a repo, or, is there other form of this settings, for example, using a directory (not repository)?

Reply:
Thanks for your suggestion. I think we need it. It can provide out-of-the-box machine learning ability for engineers who don't have machine learning knowledge, and for engineers who do have machine learning ability, they can choose their own repository or directory.

So I suggest:
1. Create a repository for out-of-the-box machine learning ability.
2. Support for setting up directories to use users' own projects.